common/beerocks/bwl/dwpal/mon_wlan_hal_dwpal.h

```diff
@@ -54,7 +56,61 @@ class mon_wlan_hal_dwpal : public base_wlan_hal_dwpal, public mon_wlan_hal {
 
     // Private data-members:
 private:
+    bool dwpal_get_scan_params_fg(sScanCfgParams &params) {
+        size_t out_len = 0;
+        if (!dwpal_nl_cmd_get(m_radio_info.iface_name, LTQ_NL80211_VENDOR_SUBCMD_GET_SCAN_PARAMS,
```

Comment:
Any reason not to change this method to return the length of the received packet? (Instead of passing `out_len`)

Reply:
done